### DISCUSSION TITLE
CK editor product/variant link

### DISCUSSION THREAD

**Discussion by lukas-jansen** [craftcms/commerce#3150] [Ideas] [answered]
I saw that the CK editor plugin has the same event to register custom link types as redactor, will products and variants be also be added by default as with redactor?
I would create a PR, but I guess a blocker will be that it requires CMS ^4.4.7

**Comment by lukeholder** [comment, marked as answer]
Thanks for your PR. https://github.com/craftcms/commerce/pull/3173

We have merged the PR and CKeditor link support will be in the next Commerce release (will be in 4.2.10).

**Reply by nfourtythree** [reply]
Version `4.2.10` of Commerce is now out with this fix.
